hypothesis-python/src/hypothesis/searchstrategy/strategies.py

```diff
@@ -20,6 +20,7 @@
 from collections import defaultdict
 from random import choice as random_choice
 
+import hypothesis._strategies as st
```

Comment:
Hmm, looks like we have an import cycle... the best option might be to simply move the `SampledFromStrategy` class from `misc.py` to this file.

Reply:
90f6d13d62186adcc277e084a21607c7a13caee5